libs/motr/motr_core_k8s_lib.py

```diff
@@ -142,13 +142,23 @@ def get_primary_cortx_node(self):
         """
         motr_client_pod = self.node_obj.get_pod_name(
             pod_prefix=common_const.CLIENT_POD_NAME_PREFIX)[1]
-        cmd = " | awk -F ' '  '/(RC)/ { print $1 }'"
+        cmd1 = " | awk -F ' '  '/(RC)/ { print $1 }'"
         primary_cortx_node = self.node_obj.send_k8s_cmd(
             operation="exec", pod=motr_client_pod, namespace=common_const.NAMESPACE,
             command_suffix=f"-c {common_const.HAX_CONTAINER_NAME} "
-                           f"-- {common_cmd.MOTR_STATUS_CMD} {cmd}",
+                           f"-- {common_cmd.MOTR_STATUS_CMD} {cmd1}",
             decode=True)
-        return primary_cortx_node.replace("data", "client")
+        data_pod_name = primary_cortx_node.split('.')[0]
```

Comment:
Abstract out k8s command complexity in another sub function.

Reply:
I think it is already available with **send_k8s_cmd** function

Reply:
the comments are not resolved .

Reply:
I want you to colocate all k8s functions in a file for motr/ probably use builder pattern to construct commands. Will help in long term.

Reply:
As discussed, I will create another ticket for making these changes.